packages/typegpu/src/index.ts

```diff
@@ -20,33 +19,65 @@ export const tgpu = {
   init,
   initFromDevice,
 
-  /** @hidden */
-  createBuffer,
-  /** @hidden */
-  read,
-  /** @hidden */
-  write,
+  resolve,
 };
 export default tgpu;
 
+export {
+  NotUniformError,
+  ResolutionError,
+} from './errors';
+export { TgpuRoot } from './core/root/rootTypes';
 export {
   isUsableAsUniform,
   isUsableAsVertex,
 } from './core/buffer/buffer';
+export {
+  isSampler,
+  isComparisonSampler,
+} from './core/sampler/sampler';
+export {
+  isSampledTextureView,
+  isStorageTextureView,
+  isTexture,
+} from './core/texture/texture';
+export {
+  isUsableAsRender,
+  isUsableAsSampled,
+} from './core/texture/usageExtension';
```

Comment:
Should we also expose `isBuffer`?

Reply:
That could be helpful as a public API, I exposed it! 🌄